controllers/mongodb_controller.go

```diff
@@ -319,6 +320,18 @@ func (r *MongoDBReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error)
 		}
 	}
 
+	// Select User to use
+	cppConfig := &corev1.ConfigMap{}
+	err = r.Client.Get(context.TODO(), types.NamespacedName{Name: "ibm-cpp-config", Namespace: instance.Namespace}, cppConfig)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
+	uid := 0
+	if cppConfig.Data["kubernetes_cluster_type"] == "cncf" {
```

Comment:
Could we check if `kubernetes_cluster_type` exists and is not equal to `ocp`?
Since in the future, `kubernetes_cluster_type` could be `tanzu` in the real use case.